maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/booterclient/lazytestprovider/OutputStreamFlushableCommandline.java

```diff
@@ -53,19 +56,27 @@ public OutputStreamFlushableCommandline()
     public OutputStreamFlushableCommandline( String[] excludedEnvironmentVariables )
     {
         this.excludedEnvironmentVariables = new ConcurrentLinkedDeque<>();
+        this.addedEnvironmentVariables = new HashSet<>(  );
```

Comment:
try to rewrite to `addedEnvironmentVariables = new HashSet<>();`
without `this` and space in constructor.

Reply:
I'm still tuning my intellij settings for surefire checkstyle...